docs/src/main/asciidoc/sqs.adoc

```diff
@@ -1231,6 +1231,8 @@ public interface MessagingMessageConverter<S> {
 
 The default header-based type mapping can be configured to use a different header name by using the `setPayloadTypeHeader` method.
 
+It is also possible not include payload type information in the header by using `doNotSendPayloadTypeHeader` method.
```

Comment:
Just two minor adjustments:
"It is also possible not to include"
and
"by using the ... method"